src/runTests.js

```diff
@@ -50,14 +50,21 @@ function getAdbPath() {
 }
 
 // Start test server, listening for test results to be posted.
-function runServer(command, dev, outputAsXml) {
+function runServer(command, dev, outputAsXml, bootTimeout) {
   server.locals.dev = dev;
   server.locals.outputAsXml = outputAsXml;
   server.listen(8082, () => {
     if (command == 'run-android') {
       runAdbReverse();
     }
     console.log(`cavy: Listening on port 8082 for test results...`);
+    setTimeout(() => {
+      if (!server.locals.appBooted) {
+        console.log("No response from Cavy.");
+        console.log("Terminating processes.");
```

Comment:
I know this is super annoying, but can we change these (and above) to single quotes like the rest of the file?